src/__tests__/metatests.js

```diff
@@ -73,3 +74,109 @@ describe('jsdom-global', () => {
     }
   });
 });
+
+describe('lolex', () => {
```

Comment:
These tests are great!

I'd like to have them added in the same commit as the implementation -- that way the implementation comes with tests from the beginning, which I think is generally desirable when there isn't a reason that would make things too complex. In particular, as you note on the tests, they make a nice set of examples for using the new API.